<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateJob</class>
 <widget class="QDialog" name="CreateJob">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_id">
         <property name="text">
          <string>Job Id</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_name">
         <property name="text">
          <string>Job Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_status">
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_cre_date">
         <property name="text">
          <string>Creation Date</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_act_date">
         <property name="text">
          <string>Activation Date</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdid_id"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_name"/>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_status"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_createDate">
         <property name="inputMask">
          <string>0000/00/00</string>
         </property>
         <property name="text">
          <string>2000/01/01</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_activateDate">
         <property name="inputMask">
          <string>0000/00/00</string>
         </property>
         <property name="text">
          <string>2000/01/01</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
